ovos_utils/system.py

```diff
@@ -56,6 +56,15 @@ def ntp_sync():
     subprocess.call('ntpd -gq', shell=True)
     subprocess.call('service ntp start', shell=True)
 
+def timesync_sync(sudo=True):
```

Comment:
Can this default to `sudo=False` to match other methods here? Also, please add a docstring like other methods

Reply:
I don't think defaulting to False would work.  The systemctl command needs sudo to stop and start.  Others are defaulting to True also
https://github.com/OpenVoiceOS/ovos-utils/blob/45b87776ae4b2556f8db502685b3d49b6b73295e/ovos_utils/system.py#L60
and others also.

I will add a docstring for this and `ntp_sync()`